ophyd/signal.py

```diff
@@ -486,9 +486,23 @@ def describe(self):
         dict
             Dictionary of name and formatted description string
         """
-        return {self.name: {'source': 'PV:{}'.format(self._read_pv.pvname),
-                            'dtype': 'number',
-                            'shape': []}}
+        desc = {'source': 'PV:{}'.format(self._read_pv.pvname), }
```

Comment:
Since I was fixing a bug with `read_configuration` and `describe_configuration` on `Devices`, it occurred to me that a better place for these would probably be in the `describe_configuration` and `read_configuration` instead - right?

Or do we want precision (etc.) read at every step?